examples/loader-rust-to-json/src/data/poker.json.rs

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env rust-script
```

Comment:
I think this line is not strictly necessary (it's ignored); one could consider it informative ("this script is supposed to run with rust-script"), but in a sense it's also confusing as it means "this script is an executable that will call this particular interpreter", when in fact it is called from framework with a (possibly) different interpreter.

Similarly, it is not necessary to have the `+x` bit set on this file. I would argue that it is (slightly) more secure in general to turn off the executable bit when not strictly required.

In other words, only the `.exe` loaders should have a shebang and be `+x` (and even then, possibly only `+x` for the user or group that runs the script, if possible).